people-and-planet-ai/weather-forecasting/README.md

```diff
@@ -0,0 +1,16 @@
+## 🌦 Weather forecasting -- _timeseries regression_
+
+> [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/GoogleCloudPlatform/python-docs-samples/blob/main/people-and-planet-ai/weather-forecasting/README.ipynb)
```

Comment:
in the "Before you Begin" section of the notebook itself there is a `repo_url` that points to your fork of the repo. Should this be updated to point to this samples repo?

Reply:
Yeah, the notebook assumes the sample is already merged. At the PR description I made a note to run this cell instead to get this branch.

```
!git clone -b ppai-weather-forecasting https://github.com/davidcavazos/python-docs-samples.git
%cd python-docs-samples/people-and-planet-ai/weather-forecasting
```